gulpfile.js

```diff
@@ -8,11 +8,12 @@ var gulp = require('gulp'),
     sourcemaps = require('gulp-sourcemaps'),
     path = require('path'),
     uglify = require('gulp-uglify'),
+    rename = require("gulp-rename"),
 
     index = './scripts/vispy.js',
     outdir = './dist/',
-    bundle = 'vispy.js',
-    outfile = 'vispy.min.js';
+    bundle = 'vispy',
+    outfile = 'vispy.js';
```

Comment:
It is.

Another PR fix the code, so that vispy.js now support standalone, AMD, etc... ie behave like a normal module without global when it's not wanted.